lib/vyasa_web/live/media_live/player.html.heex

```diff
@@ -0,0 +1,157 @@
+<div id="audio-player-container" class="sticky top-0" >
+    <!-- player -->
+    <div id="audio-player" phx-hook="AudioPlayer" class="w-full" role="region" aria-label="Player">
+      <div id="audio-ignore" phx-update="ignore">
+        <audio></audio>
+      </div>
+      <div class="bg-white dark:bg-gray-800 p-4">
+        <div class="flex items-center space-x-3.5 sm:space-x-5 lg:space-x-3.5 xl:space-x-5">
+          <div class="pr-5">
+            <div class="min-w-0 max-w-xs flex-col space-y-0.5">
+              <h2 class="text-black dark:text-white text-sm sm:text-sm lg:text-sm xl:text-sm font-semibold truncate">
+                <%= if @playback && @playback.medium, do: @playback.medium.title, else: raw("&nbsp;") %>
+              </h2>
+              <p class="text-gray-500 dark:text-gray-400 text-sm sm:text-sm lg:text-sm xl:text-sm font-medium">
+                <%= if @playback && @playback.medium, do: "Stub artist", else: raw("&nbsp;") %>
+              </p>
+            </div>
+          </div>
+          <.progress_bar
+            id="player-progress"
+            max={@playback && @playback.medium && @playback.medium.duration || 100}
+            value={@playback && @playback.elapsed || 0}
+          />
+          <div
+            id="player-info"
+            class="text-gray-500 dark:text-gray-400 flex-row justify-between text-sm font-medium tabular-nums"
+            phx-update="ignore"
+          >
+            <div id="player-time"></div>
+            <div id="player-duration"></div>
+          </div>
+        </div>
+      </div>
+      <div class="bg-gray-50 text-black dark:bg-gray-900 dark:text-white px-1 sm:px-3 lg:px-1 xl:px-3 grid grid-cols-5 items-center">
+          <div class="mx-auto flex"></div>
+          <!-- prev -->
+          <button
+            type="button"
+            class="sm:block xl:block mx-auto scale-75"
+            phx-click={js_prev()}
```

Comment:
prev and next are to be hidden before merging this in 


since it's not handled now , the view will crash

Reply:
done. 

as for the view crash, I'll let it be for now since it's going to be worked on in the immediate next PR